Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBTransportBaseClient.m

```diff
@@ -64,13 +64,15 @@ - (NSDictionary *)headersWithRouteInfo:(NSDictionary<NSString *, NSString *> *)r
                        byteOffsetStart:(NSNumber *)byteOffsetStart
                          byteOffsetEnd:(NSNumber *)byteOffsetEnd {
   NSString *routeStyle = routeAttributes[@"style"];
-  NSString *routeHost = routeAttributes[@"host"];
-  NSString *routeAuth = routeAttributes[@"auth"];
+
+  NSString *routeAuthStr = routeAttributes[@"auth"];
+  // routeAuths is one of user|team|app|noauth|app, user
+  NSArray<NSString *> *routeAuths = [routeAuthStr componentsSeparatedByString:@", "];
```

Comment:
nit: use comma without whitespace as separator may be more flexible. I don't think there is anything force the white space between auth types, either "app, user" and "app,user" may be possible value of the auth attribute.

Reply:
Updated.